### ISSUE TITLE
Questions about experiment setting.

### ISSUE THREAD

**Issue by hlqwerty** [yuhuayc/da-faster-rcnn#7]
Hi!
Nice work! I want to have a try on cityscape foggy dataset, but I do not find the exact experiment setting in the paper. Do you take cityscape training dataset as source and take foggy validation dataset as target? And the quantitative result is computed on foggy validation dataset?
By the way, I find that the instance label of cityscape test is full of 3. So there is actually no instance label for cityscape test dataset?
Thanks.

**Comment by yuhuayc** [comment]
Hi, thanks for the questions.

An example is provided in the repo for foggy dataset. Yes, we take cityscape training dataset as source and take foggy validation dataset as target, and the quantitative result is computed on foggy validation dataset.

Not sure what's exactly the question. But cityscapes dataset do has instance label, and we use the instance label for detection.

Please let me know if there is something unclear.
